notebooks/MAST/spectraldb_and_specviz/spectraldb_and_specviz.ipynb

```diff
@@ -0,0 +1,919 @@
+{
```

Comment:
<p>There's no strict standard here, but I think what you have written as <code>SpectralDB</code>  is good.   For the viz tools names though, they are typically written as all lowercase except for the first letter, i.e. <code>Jdaviz</code> or <code>Specviz</code>.    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>